packages/functions/utils/lambda/cronHandler.ts

```diff
@@ -9,7 +9,7 @@ type Event =
 
 const isApiGatewayProxyEventV2 = (
   event: Event,
-): event is APIGatewayProxyEventV2 => "queryStringParameters" in event;
+): event is APIGatewayProxyEventV2 => "routeKey" in event;
```

Comment:
So this util wraps a function so that it is able to accept a cron event both from event bridge and lamdba. But I need a way to distinguish as I need the eventId.

Initially I used queryStringParameters but now I am sometimes calling lambdas without any query params and in that case this check would fail and the logic would falsely identify a lambda event as an eventbridge event.

Reply:
I see, that's a nice one.